src/jagrati/Assignment41.java

```diff
@@ -0,0 +1,21 @@
+/*
+ Assignment 41: 01-Oct-23
+WAP to print a string from 1st vowel
+Input: pqrabcgtyes
+Output: abcgtres
+ */
+
+package jagrati;
+
+public class Assignment41 {
+	void vowels(String input) {
+		int firstIndex = input.indexOf('a');
+		int lastIndex = input.lastIndexOf('s')+1;
+		String output = input.substring(firstIndex, lastIndex);
```

Comment:
You can just give firstIndex too, it will automatically take starting from first index to end of string. As here, no condition is expected for end of string